### PR TITLE
Atlas and Data Validation UI changes

### PR DESCRIPTION
**Pull request by helena-patching-tessella**

### CODE REVIEW

File: src/PublicSite/src/uk/ac/ox/zoo/seeg/abraid/mp/publicsite/web/DataValidationController.java [hunk 1]
```diff
@@ -56,17 +56,26 @@ public DataValidationController(CurrentUserService currentUserService, DiseaseSe
...
      * @return The ftl page name.
      */
-    @RequestMapping(value = GEOWIKI_BASE_URL, method = RequestMethod.GET)
+    @RequestMapping(value = DATA_VALIDATION_BASE_URL, method = RequestMethod.GET)
     public String showTab() {
         return "datavalidation/index";
     }
 
     /**
+     * Return the the content of the help text iframe on Data Validation help modal panel.
```
Comment (edwiles) [inline src/PublicSite/src/uk/ac/ox/zoo/seeg/abraid/mp/publicsite/web/DataValidationController.java]: the the

**Comment by edwiles** [comment]
LGTM

File: src/PublicSite/web/js/app/atlas/LayerSelectorViewModel.js [hunk 2]
```diff
@@ -36,5 +37,10 @@ define([
         self.selectedLayer = ko.computed(function () {
             return self.selectedRun().id ? self.selectedRun().id + "_" + self.selectedType().id : undefined;
         }, self).publishOn("active-atlas-layer");
+
+        ko.computed(function () {
+            return self.selectedType().id || "";
+        }, self).publishOn("active-atlas-type");
```
Comment (laurence-hudson-tessella) [inline src/PublicSite/web/js/app/atlas/LayerSelectorViewModel.js]: Should there be a unit test that verifies that this event is published at the correct times?

**Comment by laurence-hudson-tessella** [comment]
LGTM with missing test :+1:

**Pull request merged**